src/shared/helpers.tsx

```diff
@@ -123,3 +123,20 @@ export const renderRoutes: {
     </Switch>
   );
 };
+
+/**
+ * Returns the appropriate URL to the cover image of an issue
+ *
+ * @param issue The issue to return the cover image for
+ * @param size The size of image to link to
+ * @param type The type of image to link to
+ */
+export const coverSrc = (issue: Issue, size: number, type?: "RGB" | "LA") =>
+  issue.pdf +
+  `.COVER-${type ?? (issue.colour === "paper" ? "RGB" : "LA")}-${size}x.png`;
+
+/**
+ * Does nothing.
+ */
+/* istanbul ignore next */
+export const noop = () => {};
```

Comment:
Can't find any references to this function. What is it for?

Reply:
It's a no-op function, we have a bunch of empty functions that are causing test coverage to fail because we can't actually test them (e.g. in Auth.ts)